README.md

```
@@ -1,187 +1,381 @@
+JWT_REFRESH=<your-refresh-token-secret>
+SALT_ROUNDS=10
+OPENAI_BASE_URI=http://localhost:3001
 ```
 
+**Frontend (.env)**
+
+```env
+NEXT_PUBLIC_GRAPHQL_URL=http://localhost:8080/graphql
+```
+
+**LLM Server (.env)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)